src/govuk/components/table/_index.scss

```diff
@@ -47,4 +47,25 @@
     display: table-caption;
     text-align: left;
   }
+
+  // Modifiers that make captions look more like their equivalent headings
+
+  .govuk-table__caption--xl {
+    @include govuk-font($size: 48, $weight: bold);
+    margin-bottom: govuk-spacing(3);
```

Comment:
Do we need to also include a line-height style for each modifier?

Reply:
The `govuk-font` mixin will automatically include the `line-height` appropriate for the specified font size as defined in the typography settings (although it can be overridden by passing a specific `$line-height`)

https://github.com/alphagov/govuk-frontend/blob/9b320cece2177f3a9749d8150e250fe8978e7eef/src/govuk/settings/_typography-responsive.scss#L85-L98